notion/renderer.py

```diff
@@ -0,0 +1,241 @@
+            <div style="flex: 4 1 180px; min-height: 60px; padding: 12px 14px 14px; overflow: hidden; text-align: left;">
+               <div style="font-size: 14px; line-height: 20px; color: rgb(55, 53, 47); white-space: nowrap; overflow: hidden; text-overflow: ellipsis; margin-bottom: 2px;">{title}</div>
+               <div style="font-size: 12px; line-height: 16px; color: rgba(55, 53, 47, 0.6); height: 32px; overflow: hidden;">{description}</div>
+               <div style="display: flex; margin-top: 6px;">
+                  <img src="{icon}" style="width: 16px; height: 16px; min-width: 16px; margin-right: 6px;">
+                  <div style="font-size: 12px; line-height: 16px; color: rgb(55, 53, 47); white-space: nowrap; overflow: hidden; text-overflow: ellipsis;">{link}/div>
+               </div>
+            </div>
+            <div style="flex: 1 1 180px; min-height: 80px; display: block; position: relative;">
+               <div style="position: absolute; top: 0px; left: 0px; right: 0px; bottom: 0px;">
+                  <div style="width: 100%; height: 100%;"><img src="{cover}" style="display: block; object-fit: cover; border-radius: 1px; width: 100%; height: 100%;"></div>
+               </div>
+            </div>
+         </div>
+      </a>
+   </div>
+</div>
+"""
+
+callout_template = """
+<div style="padding: 16px 16px 16px 12px; display: flex; width: 100%; border-radius: 3px; border-width: 1px; border-style: solid; border-color: transparent; background: rgba(235, 236, 237, 0.3);">
+   <div>
+      <div role="button" style="user-select: none; cursor: pointer; display: flex; align-items: center; justify-content: center; height: 24px; width: 24px; border-radius: 3px; flex-shrink: 0;">
+         <div style="display: flex; align-items: center; justify-content: center; height: 24px; width: 24px;">
+            <div style="height: 16.8px; width: 16.8px; font-size: 16.8px; line-height: 1.1; margin-left: 0px; color: black;">{icon}</div>
+         </div>
+      </div>
+   </div>
+   <div style="max-width: 100%; width: 100%; white-space: pre-wrap; word-break: break-word; caret-color: rgb(55, 53, 47); margin-left: 8px;">{title}</div>
+</div>
+"""
+
+class BaseHTMLRenderer(BaseRenderer):
+
+	def create_opening_tag(self, tagname, attributes={}):
+		attrs = "".join(' {}="{}"'.format(key, val) for key, val in attributes.items())
+		return "<{tagname}{attrs}>".format(tagname=tagname, attrs=attrs)
+
+	def wrap_in_tag(self, block, tagname, fieldname="title", attributes={}):
+		opentag = self.create_opening_tag(tagname, attributes)
+		innerhtml = markdown2.markdown(getattr(block, fieldname))
+		return "{opentag}{innerhtml}</{tagname}>".format(opentag=opentag, tagname=tagname, innerhtml=innerhtml)
+
+	def left_margin_for_level(self, level):
```

Comment:
IMO, inline styles would really complicate things for people wanting to post-process the output HTML or stick it into a HBS template or Angular/React/Vue app.

It'd be nicer if we could either add classes and provide a tiny CSS stylesheet (to make it easier to integrate into larger websites) or leave styling completely up to the dev calling this endpoint. This might be okay for now though, as a dev can always just `Array.from(document.querySelectorAll('*')).forEach((el)=>el.setAttribute('style', ''))` after importing.